<launch>
	<node pkg="joy" type="joy_node" name="robot_joy_node" >
		<param name="dev" value="/dev/input/by-id/usb-Logitech_Logitech_Cordless_RumblePad_2-joystick" />
		<param name="autorepeat_rate" value="2.0" />
	</node>

	<node pkg="topic_tools" type="mux" name="mux_cmd_vel"
		args="/cmd_vel /nav/cmd_vel /nifti_teleop_joy/cmd_vel" />

	<node pkg="nifti_teleop" type="nifti_teleop_joy.py"
		name="nifti_teleop_joy" output="screen"/>

	<node pkg="nifti_teleop" type="nifti_teleop_helper.py"
		name="nifti_teleop_helper" />

</launch>
